monkey-runner/README.md

```diff
@@ -41,6 +44,7 @@ buildscript {
 
 ...
 
+apply plugin: 'com.novoda.android-command'
```

Comment:
I expected so, how is it going to work without it? Is it because it is defined as a dependency in the plugin?